packages/bridge/src/promisify.ts

```diff
@@ -0,0 +1,85 @@
+import type {
+    RequestIdType,
+    AnyRequestMethodName,
+    RequestId,
+    RequestParams,
+    CondoBridgeSubscriptionListener,
+    AnyResponseMethodName,
+    ResultResponseData,
+} from './types/bridge'
+
+type PromiseController = {
```

Comment:
How it differs from type Promise ?

Reply:
Promise itself is similar to async function....and inside a controller we extract `resolve` and `reject` functions and pass it `requestResolver`

Reply:
So I need not type of Promise, but its `resolve` and `reject` functions...Did not found clear solution of doing it :(